_posts/2020-07-27-config-project-settings/config-project-settings.Rmd

```diff
@@ -12,7 +12,6 @@ output:
 categories:
   - programming
   - PACTA_analysis
-preview: preview.jpg
```

Comment:
This removes the preview image -- which is fine with me but just to check: is the removal intentional?

Would you like to propose a different preview image?

Reply:
woops, that was accidental

Reply:
Ok, I'll fix that now.

Reply:
Fixed

![image](https://user-images.githubusercontent.com/5856545/88705618-689db980-d0d5-11ea-9ea8-118bbb7e09d0.png)